README.md

```diff
@@ -0,0 +1,94 @@
+ * <img src="https://design-guidelines.web.cern.ch/sites/design-guidelines.web.cern.ch/files/u6/CERN-logo.jpg" width="20"> **made at CERN**, [the place where the web was born](https://home.cern/science/computing/birth-web)!
+
+![A sneak peek of Indico](https://raw.githubusercontent.com/pferreir/indico/new-readme/sneakpeek.gif)
+
+## What does it do?
+Indico's **main features** are:
+ * a powerful and flexible **hierarchical content management** system for events;
+ * a full-blown **conference organization workflow** with:
+   - 📢 **Call for Abstracts** and **abstract reviewing** modules;
+   - 📝 flexible **registration form** creation and configuration;
+   - 💰 integration with existing **e-payment systems**;
+   - ✅ a **paper reviewing** workflow;
+   - 🗓 a **drag and drop timetable management interface**;
+   - 🎫 a simple **badge editor** with the possibility to print badges and tickets for participants;
+ * tools for **meeting management** and archival of presentation materials;
+ * a powerful **room booking** interface;
+ * integration with existing **video conferencing solutions**;
+
+A more detailed list [can be found here](https://getindico.io/features/). There is also a [video](https://www.youtube.com/watch?v=yo8rgg9dOcc)!
+
+## I just want to try it out!
+
+We've got a  [Sandbox](https://sandbox.getindico.io)!
+
+## Browser support
+
+These are the minimum versions of major browsers currently supported by Indico. We try to target all modern browsers as
+much as possible, but only issues detected on those will be considered at critical level.
+
+<!-- BROWSERS - this is all machine-generated! Don't change it. -->
+  | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png" alt="Firefox" width="24px" height="24px" /></br>Firefox | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" alt="Chrome" width="24px" height="24px" /></br>Chrome | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png" alt="Safari" width="24px" height="24px" /></br>Safari | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png" alt="Edge" width="24px" height="24px" /></br>Edge |
+  | --------- | --------- | --------- | --------- |
+  | 65 | 73 | 12 | 17 |
+<!-- ENDBROWSERS -->
+
+## Getting Indico
+
+Information on how to get the latest release can be found [at the project's web site](https://getindico.io/getting-started/).
+There are installation guides for different systems available in [the project's documentation](https://docs.getindico.io/en/stable/installation/).
+
+## Contributing
+
+Indico is the result of the collective work of more than 100 different developers, translators and usability
+specialists of many nationalities. You can be the next one - read our
+[Contribution Guide](https://github.com/indico/indico/blob/master/CONTRIBUTING.mg) if you'd like to help out.
```

Comment:
`.md`? Also `CONTRIBUTION.md` (in the PR description) vs `CONTRIBUTING.md`

Reply:
Yeah, sorry, typo :)